### PR TITLE
Component limit improvement

### PR DESCRIPTION
**Pull request by tilmanpotthof**

### CODE REVIEW

File: rules/component-limit.js [hunk 1]
```diff
@@ -17,10 +17,11 @@ module.exports = function(context) {
 
         'Program:exit': function() {
             if(components.length > limit) {
-                components.forEach(function(node) {
-                    context.report(node, 'There may be at most {{limit}} AngularJS {{component}} per file', {
+                components.slice(limit).forEach(function(node) {
```
Comment (remcohaszing) [inline rules/component-limit.js]: See discussion [here](https://github.com/remcohaszing/eslint-plugin-angular/commit/0c93a263cd9ca3cf06166b962820b03a43db47c9#diff-df725d10e0f0addf7a294326eb4b3c94R43).

Comment (tilmanpotthof) [inline rules/component-limit.js]: @remcohaszing Ok, I'll revert these changes. If the result of the discussion changes you can easily revert the revert.

**Comment by remcohaszing** [comment]
Merged manually except the last commit. I have decided I don't really mind which nodes are reported exactly, so I'll just accept whatever the package owner prefers.

**Comment by tilmanpotthof** [comment]
Allright. I had no real preferences and understood both approaches. The rule itself is way more important.

**Pull request closed without merging**